client/api.go

```diff
@@ -7,10 +7,18 @@ import (
 	v2netmap "github.com/nspcc-dev/neofs-api-go/v2/netmap"
 	rpcapi "github.com/nspcc-dev/neofs-api-go/v2/rpc"
 	"github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
+	"github.com/nspcc-dev/neofs-api-go/v2/session"
 )
 
+// sessionAPI should contain only session operations. Dividing interfaces should help to improve testability.
+type sessionAPI interface {
```

Comment:
seems like now we don't need to have a separate interface for sessions because it's used only in `neoFSAPIServer` definition, lets drop?

Reply:
I've added it in case of we need to test session separate. It may reduce amount of funcs to implement in tests. Right now it is useless, removed